[2022-09-16T23:15:46.135+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-09-16T23:15:39.068877+00:00 [queued]>
[2022-09-16T23:15:46.152+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-09-16T23:15:39.068877+00:00 [queued]>
[2022-09-16T23:15:46.153+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-16T23:15:46.154+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2022-09-16T23:15:46.154+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-16T23:15:46.173+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-09-16 23:15:39.068877+00:00
[2022-09-16T23:15:46.182+0000] {standard_task_runner.py:52} INFO - Started process 194 to run task
[2022-09-16T23:15:46.187+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'manual__2022-09-16T23:15:39.068877+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpxrmbkwfa', '--error-file', '/tmp/tmpq3vlnqae']
[2022-09-16T23:15:46.189+0000] {standard_task_runner.py:80} INFO - Job 38: Subtask local_to_gcs_task
[2022-09-16T23:15:46.272+0000] {task_command.py:371} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-09-16T23:15:39.068877+00:00 [running]> on host 6f8e2375011a
[2022-09-16T23:15:46.364+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-09-16T23:15:39.068877+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-16T23:15:39.068877+00:00
[2022-09-16T23:15:47.277+0000] {python.py:173} INFO - Done. Returned value was: None
[2022-09-16T23:15:47.295+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20220916T231539, start_date=20220916T231546, end_date=20220916T231547
[2022-09-16T23:15:47.347+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-16T23:15:47.394+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
